Miracast/P2P/MiracastP2P.cpp

```diff
@@ -437,10 +434,25 @@ void MiracastP2P::reset_WFDParameters(void)
 MiracastError MiracastP2P::discover_devices(void)
 {
     MiracastError ret = MIRACAST_FAIL;
-    std::string command, retBuffer;
+    std::string command, retBuffer,opt_flag_buffer;
     MIRACASTLOG_TRACE("Entering..");
 
     command = "P2P_FIND";
+
+#ifdef ENABLE_DELAY_FOR_MIRACAST_P2P_SCAN
```

Comment:
We dont want this logic right?

Reply:
Yes. Removed it in https://github.com/rdkcentral/rdkservices/pull/4667/commits/740e0e7e079093a2cfc24217fcb8291944223489